### ISSUE TITLE
Fix rule 124

### ISSUE THREAD

**Issue by alligator77** [voku/PHPMailer-BMH#6]
Add in rules ```$result['email'] = $match[1];```
In callback function param $email is empty without this line
```  
elseif (preg_match("/<(\S+@\S+\w)>.*\n.*mailbox unavailable/i", $body, $match)) {
    $result['rule_cat'] = 'unknown';
    $result['rule_no'] = '124';
    /* HERE */
    $result['email'] = $match[1];
  }
```

**Comment by alligator77** [comment]
Created Pull request

**Comment by voku** [comment]
thx for the bug-report + fix -> fixed in v5.4.4
